src/pages/Home/Goals/index.tsx

```diff
@@ -0,0 +1,32 @@
+import * as React from "react";
+import { Container, Item, Title, Box, Bullet } from "./styles";
+
+interface Props {}
+
+const Goals: React.SFC<Props> = (props) => {
+  return (
+    <Container>
+      <Title>Objetivos</Title>
+      <Box>
+        <Item>
+          <Bullet>●</Bullet> Desenvolver projetos para evolução da comunidade
```

Comment:
Just a suggestion, when you want add a symbols or anything like this dot, you can use the unicodes/charsets, 'cause can be that other Browser not render this symbol correctly. For example, for this dot, you can use `&bull;` (You can see more symbols on this link:https://www.w3schools.com/charsets/ref_utf_punctuation.asp). Anyway, you Didi a good job 😄👌🏻

Reply:
Didn't know that!

Reply:
Didn't know that. I'll correct this on #13.